### ISSUE TITLE
paperless-ngx update failed with pip install -r

### ISSUE THREAD

**Issue by ChristophCaina** [tteck/Proxmox#2399]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

I am running an debian-11 based container which was once installed with the Paperless-ngx script from this website.
Updates were working well in the past - today, I wanted to update from 1.17.4 to 2.4.2 but the install script is failing repeatingly with the following error on `pip install -r requirements.txt &> /dev/null`

### Which Linux distribution are you employing?

Debian 11

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

```
    ____                        __                                     
   / __ \____ _____  ___  _____/ /__  __________    ____  ____ __  __
  / /_/ / __ `/ __ \/ _ \/ ___/ / _ \/ ___/ ___/___/ __ \/ __ `/ |/_/
 / ____/ /_/ / /_/ /  __/ /  / /  __(__  |__  )___/ / / / /_/ />  <  
/_/    \__,_/ .___/\___/_/  /_/\___/____/____/   /_/ /_/\__, /_/|_|  
           /_/                                         /____/        
 
 ✓ Stopped Paperless-ngx
 - Updating to v2.4.2  |
[ERROR] in line 86: exit code 0: while executing command pip install -r requirements.txt &> /dev/null
```

**Comment by tteck** [comment]
What is the output from `python3 --version`?

**Comment by ChristophCaina** [comment]
root@paperless-ngx:~# python3 --version
Python 3.9.2

**Comment by tteck** [comment]
I just successfully created a new Debian 11 Paperless-ngx LXC. Then I updated an already up to date app successfully, so it still works with Debian 11. 
Maybe a
```
cd /opt/paperless
pip install -r requirements.txt
```
will show the issue.
That's about all that I can help with.

**Comment by tteck** [comment]
That is a 14 version jump! 1.17.4 is from Sep 1, 2023

**Comment by tteck** [comment]
I would suggest a fresh Debian 12 (the default Linux distribution) install.

As this issue is unrelated to the script, it is considered closed.

**Comment by ChristophCaina** [comment]
thanks.
I also wanted to upgrade to debian12, which also caused some issues (probably due to the postgress sql)

I did try the pip install -r requirements.txt you've suggested, and it seems that pip needed to be updated.
```
[notice] A new release of pip available: 22.3.1 -> 23.3.2
[notice] To update, run: python3 -m pip install --upgrade pip
```

it seems, that after upgrading pip the script was running successfully
